Samples/2.0/ChatRoom/OrleansClient/Program.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using GrainInterfaces;
+using GrainInterfaces.Model;
+using Microsoft.Extensions.Logging;
+using Orleans;
+using Orleans.Configuration;
+using Orleans.Hosting;
+using Orleans.Runtime;
+using Microsoft.Extensions.DependencyInjection;
+using Orleans.Streams;
+using Utils;
+
+namespace OrleansClient
+{
+	class Program
+	{
+        //To make this sample simple
+        //In this sample, one client can only join one channel, hence we have a static variable of one channel name.
```

Comment:
We can as easily allow multiple clients to join the same channel.

Reply:
yes multiple client can join the same channel, which is the point of a chat room. it is just one client cannot join multiple channel at the same time. We can technically do that, but without proper UI, it would look confusing from the console. That's why I designed it this way.